maven-script/maven-plugin-tools-beanshell/src/main/java/org/apache/maven/tools/plugin/extractor/beanshell/BeanshellMojoDescriptorExtractor.java

```diff
@@ -131,6 +131,7 @@ private MojoDescriptor createMojoDescriptor( String basedir, String resource, Pl
             throw new InvalidPluginDescriptorException( "Error scanning beanshell script", evalError );
         }
 
+        // FIXME: convert javadocs
```

Comment:
necessary to convert javadocs -> xhtml for deprecated BSH extractor?

Reply:
I left the legacy handling in place, i.e. convert inline javadoc tags afterwards to HTML (but only for legacy extractors).